wlm_server/cache/channel.py

```diff
@@ -25,6 +30,11 @@ def _load(self):
         settings = Setting.objects.order_by('channel', '-created_at').distinct('channel')
         for setting in settings:
             self._channel_to_setting[setting.channel.channel] = setting
+        locks = (Lock.objects.order_by('channel', '-started_at').distinct('channel'))
+        now = timezone.now()
+        for lock in locks:
+            if lock.expires_at > now:
+                self._channel_to_lock[lock.channel.channel] = lock
```

Comment:
As far as I understand, this runs only once at the beginning. How does it check the expiration while running? For example, if the `release_lock()` in `views.py` is called, then the `lock.expires_at` is updated to `now`. Then is it removed from `_channel_to_lock` dictionary immediately, so `util.verify_channel_access()` can correctly check the lock acquisition?

Reply:
Still, #20 says that it implements the manual release of the locks. However, I think `release_lock()` also just changes `lock.expires_at` without modifying `_channel_to_lock`. If this is what you intended, I am okay with it.

Reply:
Oh, now I understand what your point is. I think the lock is correctly deleted from the cache as follows.

1. One calls `release_lock()`, then the modified lock is saved through `lock.save()`.
2. `handle_model_save()` is called because it is connected through the `post_save` signal.
3. In this function, `channel_cache.delete_lock()` deletes the corresponding lock from the cache.

Does it make sense?